.ci/deps.cabal.sh

```diff
@@ -0,0 +1,13 @@
+set -e
+
+# cabal-install 1.22 is the minimum tested version
+cabal --version
+
+cabal update
+
+rm $HOME/.cabal/config
```

Comment:
This maybe unnecessary.
If the version of cabal-install is too low, and needs re-installing here, the config could be stale and will make the new cabal less functional.